src/Inkluzitron/Modules/MockingModule.cs

```diff
@@ -27,24 +31,54 @@ public async Task MockAsync(params string[] strings)
                     return;
                 }
 
+                hasReferencedMsg = true;
                 message = Context.Message.ReferencedMessage.ToString().ToLower();
             }
 
             var newString = "";
-            var toUpper = false;
+            var rnd = new Random(int.Parse(DateTimeOffset.Now.ToUnixTimeSeconds().ToString()));
```

Comment:
Na co vytváříš novou instanci Random, když jde vytáhnout přes Dependency Injection? 🤔

Reply:
Vubec netusim, ze neco takoveho je tbh 😆 Fixnes to ty, nebo mi nejak osvetlis jak na to? 🤔

Reply:
Já to pak někdy fixnu. Necháme to tak